test/cmdlineTests/function_debug_info/output

```diff
@@ -0,0 +1,43 @@
+{
+  "contracts":
+  {
+    "function_debug_info/input.sol:C":
+    {
+      "function-debug": {},
```

Comment:
Maybe you could add a constructor in the input and do some calls from it to make sure this works too?